### PR TITLE
chore(web): introduce i18nnext

### PR DESCRIPTION
**Pull request by issmail-basel**

### CODE REVIEW

File: web/src/i18n.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import * as i18n from "i18next";
+import { initReactI18next } from "react-i18next";
+
+import en from "../translations/en.json";
+import de from "../translations/ja.json";
```
Comment (rot1024) [inline web/src/i18n.ts:5]:
```suggestion
import ja from "../translations/ja.json";
```

**Review by rot1024 — APPROVED** [review]

**Comment by rot1024** [comment]
- Add a npm script yarn run i18n to extracts translations and adds  them to the json files
- CI checks all texts are translated (if no, CI fails)
- Change yarn gen:gql to yarn gql because I'm tired of typing every single gen:
- Use yaml instead of json
- Add i18next-browser-languagedetector

File: web/src/App.tsx [hunk 2]
```diff
@@ -3,6 +3,7 @@ import RootPage from "@reearth-cms/components/pages/Authentication/RootPage";
 import { BrowserRouter as Router, useRoutes } from "react-router-dom";
 
 import { Provider as GqlProvider } from "./gql";
+import { Provider as I18nProvider } from "./i18n";
```
Comment (KaWaite) [inline web/src/App.tsx:6]: Please remember to use the @reearth-cms alias

**Review by KaWaite — APPROVED** [review]
Nice, thanks for keeping it in-line with Inoue-san's PR for reearth-web as well!

**Pull request merged**